features/search/components/NavigationChatButton.tsx

```diff
@@ -27,12 +24,16 @@ export function NavigationChatButton({
   groupMode,
   addToGroup,
 }: NavigationChatProps) {
-  const styles = useStyles();
+  const { theme } = useAppTheme();
+
   const [loading, setLoading] = useState(false);
   const profile = useProfilesStore(
     useShallow((s) => getProfile(address, s.profiles))
   );
   const preferredName = getPreferredName(profile?.socials, address);
+  const isCurrentUser =
+    address.toLowerCase() === currentAccount().toLowerCase();
```

Comment:
Not sure if we have one but we should have a function somewhere that does `isCurrentUser(address: string)`

Reply:
Done :)